src/index.html

```diff
@@ -22,6 +22,376 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo-link"
+      >
+        <img
+          alt="Logo Moyo"
+          src="images/logo.png"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__menu menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
```

Comment:
Please remember to add empty lines between sibling elements, it is requested in [Code Styles Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Add%20empty%20lines%20between%20multiline%20sibling%20blocks%20of%20HTML)